src/facade/redis_parser.cc

```diff
@@ -363,13 +366,17 @@ auto RedisParser::ParseArg(Buffer str) -> ResultConsumed {
       return res;
     }
 
+    if (len > absl::GetFlag(FLAGS_max_bulk_len)) {
+      LOG_EVERY_T(WARNING, 1) << "Threshold reached with bulk len: " << len;
```

Comment:
on the same note, please mention the flag here. and please cache it in a variable.